classes/classes/Items/Mutations.as

```diff
@@ -1265,6 +1253,21 @@
 				player.skinDesc = "fur";
 				player.underBody.restore(); // Restore the underbody for now
 			}
+			// Hooves - Tail and not taur
+			if (player.lowerBody != LOWER_BODY_TYPE_HOOFED && !player.isTaur() && player.tailType == TAIL_TYPE_HORSE && changes < changeLimit && rand(5) == 0) {
+				changes++;
+				if (player.lowerBody == LOWER_BODY_TYPE_HUMAN) outputText("\n\nYou stagger as your feet change, curling up into painful angry lumps of flesh.  They get tighter and tighter, harder and harder, until at last they solidify into hooves!", false);
+				if (player.lowerBody == LOWER_BODY_TYPE_DOG) outputText("\n\nYou stagger as your paws change, curling up into painful angry lumps of flesh.  They get tighter and tighter, harder and harder, until at last they solidify into hooves!", false);
+				if (player.lowerBody == LOWER_BODY_TYPE_NAGA) outputText("\n\nYou collapse as your sinuous snake-tail tears in half, shifting into legs.  The pain is immense, particularly in your new feet as they curl inward and transform into hooves!", false);
+				//Catch-all
+				if (player.lowerBody > LOWER_BODY_TYPE_NAGA) outputText("\n\nYou stagger as your " + player.feet() + " change, curling up into painful angry lumps of flesh.  They get tighter and tighter, harder and harder, until at last they solidify into hooves!", false);
```

Comment:
Using inequations for unsorted enums is considered bad practice. You should either use a switch....case or if...else if...else here.

if...else if....else:
```as3
				if (player.lowerBody == LOWER_BODY_TYPE_HUMAN)
					outputText("\n\nYou stagger as your feet change, curling up into painful angry lumps of flesh.  They get tighter and tighter, harder and harder, until at last they solidify into hooves!");
				else if (player.lowerBody == LOWER_BODY_TYPE_DOG)
					outputText("\n\nYou stagger as your paws change, curling up into painful angry lumps of flesh.  They get tighter and tighter, harder and harder, until at last they solidify into hooves!");
				else if (player.lowerBody == LOWER_BODY_TYPE_NAGA)
					outputText("\n\nYou collapse as your sinuous snake-tail tears in half, shifting into legs.  The pain is immense, particularly in your new feet as they curl inward and transform into hooves!");
				else //Catch-all
					outputText("\n\nYou stagger as your " + player.feet() + " change, curling up into painful angry lumps of flesh.  They get tighter and tighter, harder and harder, until at last they solidify into hooves!");
```
switch...case:
```as3
				switch (player.lowerBody) {
					case LOWER_BODY_TYPE_HUMAN:
						outputText("\n\nYou stagger as your feet change, curling up into painful angry lumps of flesh.  They get tighter and tighter, harder and harder, until at last they solidify into hooves!");
						break;

					case LOWER_BODY_TYPE_DOG:
						outputText("\n\nYou stagger as your paws change, curling up into painful angry lumps of flesh.  They get tighter and tighter, harder and harder, until at last they solidify into hooves!");
						break;

					case LOWER_BODY_TYPE_NAGA:
						outputText("\n\nYou collapse as your sinuous snake-tail tears in half, shifting into legs.  The pain is immense, particularly in your new feet as they curl inward and transform into hooves!");
						break;

					default:
						outputText("\n\nYou stagger as your " + player.feet() + " change, curling up into painful angry lumps of flesh.  They get tighter and tighter, harder and harder, until at last they solidify into hooves!");
				}
```

PS: I've typed this into the textarea here, meaning: Its not tested. So, although the examples look good to me, make sure you try to compile it before hitting the push-button.